docs/user/library/copy/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Copy and Reuse
+show-last-update-time: true
+---
+
+# Why copying a capsule?
+
+Copying a capsule in your space allows you to reuse existing content without altering the original, making it easy to build on proven ideas or customize the content to fit your specific needs or audience. Additionally, copying gives you the flexibility to use it privately, so you can experiment, test changes, or refine the content in your own workspace without affecting the public version.
+
+:::warning
+Before copying and reusing a capsule, it’s important to check the Creative Commons (CC) license associated with it. Some capsules may come with restrictions, ignoring these terms can lead to copyright issues or misuse of someone else’s work. Always review the license details and respect the original creator’s permissions to ensure your use complies with the intended sharing guidelines.
+:::
+
+# How to copy a capsule
+
+1. Once you’ve found a capsule you want to copy, click on the three dots menu (⋮), then click on **Copy**.
+
+![Copy Button](library_copy_button.png)
+
+2. You will be redirected to Builder. If you are logged out, a warning will be asked to login.
+
+![Copy warning logged out](copy_logged_out.png)
+
+3. Once logged in, you can select where you want to copy this capsule. Note that it can take a few instants before everything as been copied depending on the size of the capsule.
```

Comment:
```suggestion
3. Once logged in, you can select where you want to copy this capsule. Note that it can take a few instants before everything has been copied depending on the size of the capsule.
```